src/main.rs

```diff
@@ -93,89 +104,137 @@ fn main() -> Result<(), Error> {
     // Set up our two websocket connections on the same event loop
     let mut eloop = tokio_core::reactor::Core::new()?;
     let handle = eloop.handle();
+
     let home_ws = web3::transports::WebSocket::with_event_loop(&settings.relay.homechain.wsuri, &handle)
-        .map_err(SyncFailure::new)?;
+        .map_err(SyncFailure::new)
+        .unwrap();
+
```

Comment:
Why unwrap over the `?`?

Reply:
Oh nice find, I was getting errors requiring that `?` can only be used in a function returning a Result or Option at some point but it's not an issue anymore, I'll throw that back in. :+1: